src/zenml/integrations/__init__.py

```diff
@@ -58,9 +58,9 @@
 from zenml.integrations.scipy import ScipyIntegration  # noqa
 from zenml.integrations.seldon import SeldonIntegration  # noqa
 from zenml.integrations.sklearn import SklearnIntegration  # noqa
-from zenml.integrations.skypilot import SkypilotAWSIntegration  # noqa
-from zenml.integrations.skypilot import SkypilotGCPIntegration  # noqa
-from zenml.integrations.skypilot import SkypilotAzureIntegration  # noqa
+from zenml.integrations.vm_aws import SkypilotAWSIntegration  # noqa
+from zenml.integrations.vm_gcp import SkypilotGCPIntegration  # noqa
+from zenml.integrations.vm_azure import SkypilotAzureIntegration  # noqa
```

Comment:
Why are we going back to the `vm_` prefix? Didn't we just change the integation names over to `skypilot_aws`? They should be consistent, I think.